app/reducers/registrations.js

```diff
@@ -34,13 +34,21 @@ export default createEntityReducer({
         if (!registration) {
           return state;
         }
+        let paymentErrorData = {};
```

Comment:
Do you use this anywhere, or did you cherry-pick half my commit?

Reply:
I overlooked this file when I cherrypicked your commit, but I think I will just remove the changes here, since the errors are not used anywhere. And we can add more error handling later if we need to.